src/XenCert/StorageHandler.py

```diff
@@ -1,5 +1,4 @@
-# Copyright (c) 2005-2022 Citrix Systems Inc.
```

Comment:
These are similar, need not to remove as my understanding.

Reply:
I have recovered it